sample.org

```diff
@@ -316,6 +316,53 @@ Last but not least, you can search for headlines with defined properties:
 
 This filters headlines having a property *blocked_by* (with any value) and a property *assignee* with a value containing =nobody= or =none=.
 
+You can also search time ranges on headers with planning items (SCHEDULED and DEADLINE), plain active timestamps or clocked work time.
+
+The time related filters are:
+
+- =scheduled= (shorthand =sched=)
+- =deadline= (shorthand =dead=)
+- =clock=
+- =date=
+
+=date= is a shorthand to search for planning items (SCHEDULED and DEADLINE) and plain active timestamps at the same time.
+
+These time related filters work on time ranges indicated by =START..END= where =START= and =END= support the following units:
+
+- =h=: Hours
+- =d=: Days
+- =w=: Weeks
+- =m=: Months
+- =y=: Years
+
+=START= and =END= can both be omitted to search for an infinite timespan into the past or future. Both can be used with digits to quantify the amount (i.e. =2w= for two weeks). For both, the digit can be omitted, so =1w= is equivalent to =w=.
```

Comment:
Sorry if I'm wrong, but I had in mind that `w` was "from beginning of this week" (and "to end of this week").

Reply:
As it currently is implemented `date:1w` is equivalent to `date:..1w` or in words 'from now to one week in the future'.
`date:..w` is 'from now until the end of the week'
`date:w` is the current week (start to finish).

Reply:
Thank you for your feedback! I have changed the documentation accordingly: https://github.com/200ok-ch/organice/pull/505/commits/4216e1655776143ec23cb36b9926a28b124d2d0d

I've also added information on negating filters.

Reply:
It's only been two days since I've asked for feedback, but I assume you're good with the changes, because both of you have been active in other issues/prs since(;

I'll go ahead, add the attribution to the changelog and will merge to `master`. If you do find something that want changed in the documentation, we can change it anytime!